presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -21,10 +21,418 @@
 
 namespace facebook::presto {
 
+#define SYS_CONF_BASE_PROPS(V)                                                \
+  /* Setting this to 'true' makes configs modifiable via server operations.*/ \
```

Comment:
I'm not familiar with this code, but I wonder where did these come from? In general, comments on public methods / variables should use ///

Reply:
Yea, I notice the `///` for the general use. However, inside a macro, it's actually a very long string. Therefore, not able to use `///` for the comment. It will comment out all of the following lines.